MonoGame.Framework/Input/MouseState.cs

```diff
@@ -29,25 +29,62 @@ public struct MouseState
         /// <param name="xButton1">XBUTTON1's state.</param>
         /// <param name="xButton2">XBUTTON2's state.</param>
         /// <remarks>Normally <see cref="Mouse.GetState()"/> should be used to get mouse current state. The constructor is provided for simulating mouse input.</remarks>
-		public MouseState (
-			int x,
-			int y,
-			int scrollWheel,
-			ButtonState leftButton,
-			ButtonState middleButton,
-			ButtonState rightButton,
-			ButtonState xButton1,
-			ButtonState xButton2)
-		{
-			_x = x;
-			_y = y;
-			_scrollWheelValue = scrollWheel;
-			_leftButton = leftButton;
-			_middleButton = middleButton;
-			_rightButton = rightButton;
+        public MouseState(
+            int x,
+            int y,
+            int scrollWheel,
+            ButtonState leftButton,
+            ButtonState middleButton,
+            ButtonState rightButton,
+            ButtonState xButton1,
+            ButtonState xButton2) : this(
```

Comment:
Because this is a simple struct, and structs have special rules regarding all fields must be set, I would duplicate the setting of the fields in this constructor rather than calling the other constructor using `this`.